readthedocs/doc_builder/config.py

```diff
@@ -0,0 +1,114 @@
+
+from readthedocs_build.config import (ConfigError, BuildConfig,
+                                      load as load_config)
+
+
+class ConfigWrapper(object):
```

Comment:
We can always move as `readthedocs-build` matures, if that makes sense. This seems like a fine approach for now